docs/products/postgresql/howto/monitor-database-with-datadog.md

```diff
@@ -84,8 +81,8 @@ Monitoring for your service.
 
 ## Related pages
 
--   Learn more about
-    [Datadog and Aiven](/docs/integrations/datadog).
--   Learn more about [Datadog Deep Database
-    Monitoring](https://www.datadoghq.com/product/database-monitoring/)
-    from their product page.
+- Learn more about [Datadog and Aiven](/docs/integrations/datadog).
+- [Expose PgBouncer statistics to Datadog](/docs/products/postgresql/howto/pgbouncer-stats-datadog).
+- Learn more about
+  [Datadog Deep Database Monitoring](https://www.datadoghq.com/product/database-monitoring/)
```

Comment:
This is a minor thing but Datadog dropped the term "Deep" from their offering name (it is only visible in the main page headline but not referred to in any other place) so I recommend changing this to "Database monitoring with Datadog" to keep in line with the title of our own doc (https://aiven.io/docs/products/postgresql/howto/monitor-database-with-datadog).